src/components/search-form/search-form.tsx

```diff
@@ -0,0 +1,25 @@
+import styles from '../movie-list-page/styles.module.scss'
+import { Search } from '../../ui'
+import { useSearchParams } from 'react-router-dom'
+import { useCallback } from 'react'
+
+export const SearchForm = () => {
+  const [searchQuery, setSearchQuery] = useSearchParams()
```

Comment:
A confusing name for state, it's not only query, but all params:

```suggestion
  const [searchParams, setSearchParams] = useSearchParams()
```